hs_core/tests/api/views/test_replicate_bag_to_user_zone.py

```diff
@@ -0,0 +1,57 @@
+import json
+
+from django.test import TestCase, RequestFactory
+from django.contrib.auth.models import Group
+from django.core.urlresolvers import reverse
+
+from rest_framework import status
+
+from hs_core import hydroshare
+from hs_core.views import rep_res_bag_to_irods_user_zone
+from hs_core.testing import TestCaseCommonUtilities
+
+
+class TestReplicateBagToUserZone(TestCaseCommonUtilities, TestCase):
+    def setUp(self):
+        super(TestReplicateBagToUserZone, self).setUp()
+        if not super(TestReplicateBagToUserZone, self).is_federated_irods_available():
+            return
+
+        self.group, _ = Group.objects.get_or_create(name='Hydroshare Author')
+        self.username = 'john'
+        self.password = 'jhmypassword'
+        self.user = hydroshare.create_account(
+            'john@gmail.com',
+            username=self.username,
+            first_name='John',
+            last_name='Clarson',
+            superuser=False,
+            password=self.password,
+            groups=[]
+        )
```

Comment:
I think ```super(TestUserZoneIRODSFederation, self).create_irods_user_in_user_zone()``` needs to be called to ensure the user has a corresponding user account in user zone. Refer to https://github.com/hydroshare/hydroshare/blob/develop/hs_core/tests/api/native/test_irods_user_zone_federation.py for details.